### ISSUE TITLE
MultilineMethodCallIndentation should accept IndentationWidth when EnforcedStyle: aligned

### ISSUE THREAD

**Issue by hampei** [rubocop-hq/rubocop#4053]
The MultilineMethodCallIndentation cop give an error when you specify a IndentationWidth while enforcing the aligned style.

```
Error: The `Style/MultilineMethodCallIndentation` cop only accepts an `IndentationWidth` configuration parameter when `EnforcedStyle` is `indented`.
```

## Expected behavior

```
Style/MultilineMethodCallIndentation:
  IndentationWidth: 4
  EnforcedStyle: aligned
```

allows code like

```
wegotsomereallylongvariableorsequence
    .something
    .and_then

bar.foo
   .something
```

## Actual behavior

Errors when you try to set it, even though the Default indentation width is still enforced in some cases.

with configuration
```
Style/MultilineMethodCallIndentation:
  EnforcedStyle: aligned
```
Both these are accepted

```
wegotsomereallylongvariableorsequence
  .something
  .and_then

bar.foo
   .something
```

But 

```
wegotsomereallylongvariableorsequence
    .something
    .and_then
```

gives you 

```
Style/MultilineMethodCallIndentation: Use 2 (not 4) spaces for indenting an expression spanning multiple lines.
```

While the configuration

```
Style/MultilineMethodCallIndentation:
  IndentationWidth: 4
  EnforcedStyle: aligned
```

gives you 

```
Error: The `Style/MultilineMethodCallIndentation` cop only accepts an `IndentationWidth` configuration parameter when `EnforcedStyle` is `indented`.
```

## RuboCop version

```
✗ rubocop -V
0.47.1 (using Parser 2.4.0.0, running on ruby 2.3.3 x86_64-darwin15)
```

**Comment by MrEmelianenko** [comment]
Guys, from what time this indentation is wrong? Two spaces after parent method:

```ruby
url = params
        .permit(:controller_name, :action_name, :q)
        .merge(searchable_type: searchable_type)
```

But indentation without assignment is correct.
```ruby
params
  .permit(:controller_name, :action_name, :q)
  .merge(searchable_type: searchable_type)
```

Is this bug?

**Comment by jonas054** [comment]
The validation that rejects the `IndentationWidth` setting was added by @alexdowad here:
https://github.com/bbatsov/rubocop/pull/2602/commits/8e982388dae9b2e4304eb190ee4a9202e6f3ddcb

It seems to me that the cop actually uses its `IndentationWidth` setting in the `EnforcedStyle: aligned` case when it falls back on indentation because no suitable base column for alignment was found.

**Comment by bbatsov** [comment]
I'm closing this ticket due to no recent activity. Feel free to re-open it if you ever come back to it.
